packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -602,12 +601,6 @@ export default function NavigationSubmenuEdit( {
 							onClose={ () => setIsLinkOpen( false ) }
 							anchorRef={ listItemRef.current }
 						>
-							<KeyboardShortcuts
-								bindGlobal
-								shortcuts={ {
-									escape: () => setIsLinkOpen( false ),
-								} }
-							/>
```

Comment:
This is useless code. `Popover` already handles escape and calls the `onClose` prop.